sparql-ns.ttl

```diff
@@ -0,0 +1,403 @@
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-round> ;
+    .
+
+sparql:ceil rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-ceil> ;
+    .
+
+sparql:floor rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-floor> ;
+    .
+
+# Section: Functions on datetimes
+
+sparql:now rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-now> ;
+    .
+
+sparql:year rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-year> ;
+    .
+
+sparql:month rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-month> ;
+    .
+
+sparql:day rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-day> ;
+    .
+
+sparql:hours rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-hours> ;
+    .
+
+sparql:minutes rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-minutes> ;
+    .
+
+sparql:seconds rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-seconds> ;
+    .
+
+sparql:timezone rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-timezone> ;
+    .
+
+sparql:tz rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-tz> ;
+    .
+
+# Section: Functions on triple terms
+
+sparql:triple rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-triple> ;
+    .
+
+sparql:subject rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-subject> ;
+    .
+
+sparql:predicate rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-predicate> ;
+    .
+
+sparql:object rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-object> ;
+    .
+
+
+# Section: Hash functions
+
+sparql:md5 rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-md5> ;
+    .
+
+sparql:sha1 rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-sha1> ;
+    .
+
+sparql:sha256 rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-sha256> ;
+    .
+
+sparql:sha384 rdf:type sparql:Function ;
+    dc:description "" ;
```

Comment:
This function computes the SHA-384 hash of the lexical form of a string, returning the resulting hexadecimal string.

Reply:
```suggestion
    dc:description "This function computes the SHA-384 hash of the lexical form of a string, returning the resulting hexadecimal string." ;
```